packages/amplication-client/src/Workspaces/WorkspaceHeader/WorkspaceHeader.tsx

```diff
@@ -103,7 +109,11 @@ const WorkspaceHeader: React.FC = () => {
   }, [history, apolloClient]);
 
   const onNotificationClick = useCallback((message: IMessage) => {
-    // your logic to handle the notification click
+    trackEvent({
+      eventName: AnalyticsEventNames.ClickNotificationMessage,
+      messageType: "now-buildSucceeded",
```

Comment:
did you mean "new-buildSucceeded"? Or perhaps I don't understand the context for "now" here?

Reply:
it came from paz